static/styles/app_components.css

```diff
@@ -191,6 +191,11 @@
         }
     }
 
+    a:hover,
+    a:focus {
+        text-decoration-line: none;
+    }
```

Comment:
I think these are incorrectly not scoped to just new-style elements?  We don't want to remove this from all links in the app.

Reply:
so should I apply it on the "forgot link" password?

Reply:
No, you should scope it to buttons with this implementation approach, not all links.

Reply:
should I change the <a> tag to <button>  for invite-more users or add a class for those particular buttons to have this property?